File: aspnet-async/src/AspNetAsyncBenchmark/Controllers/PlainTextController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspNetAsyncBenchmark.Controllers
{
	public class PlainTextController 
		: Controller
	{
		public ActionResult Index()
		{
			return Content("Hello, World!", "text/plain");
		}
	}
}

File: frameworks/PHP/kumbiaphp/bench/app/controllers/plaintext_controller.php
<?php

/**
 * Controller por defecto si no se usa el routes
 *
 */
class PlaintextController extends AppController
{

    public function index()
    {
        View::select(null, null);
        header('Content-Type: text/plain');
        echo 'Hello, World!';
    }
}
